tracer/src/Datadog.Tracer.Native/cor_profiler.cpp

```diff
@@ -1389,7 +1389,20 @@ void CorProfiler::InitializeProfiler(WCHAR* id, CallTargetDefinition* items, int
 
     if (size > 0)
     {
-        InternalAddInstrumentation(id, items, size, false);
+        InternalAddInstrumentation(id, items, size, false, true);
+    }
+}
+
+void CorProfiler::RemoveCallTargetDefinitions(WCHAR* id, CallTargetDefinition* items, int size)
+{
+    auto _ = trace::Stats::Instance()->InitializeProfilerMeasure();
```

Comment:
We should create a new Stats for this, all measurements are printed on profiler shutdown.

Reply:
Removed it on e3ca95392f32b1632ba61e0b78f15421b385ab85

Reply:
I meant to add a new Stats 🤣. To no block this, we can add it in another PR. But I'm interesting on tracking the time spent on reverting.